chizzydavid/epic-mail
# Fix ui

### What does this Pull Request Do
**Added Scroll into view functionality for some UI elements**


### Specific steps carried out to accomplish this task
- Added scroll into view feature when the UI element displaying feedback in my Sign Up form is triggered.

### How can this feature be tested
- In the sign-up form, enter invalid input to trigger the feedback element and then submit. The response element should scroll into view nicely.
